modules/installation/pages/install-intro.adoc

```diff
@@ -21,4 +22,6 @@ This is the default method of installation.
 
 endif::[]
 
+
+// REMARK this para needs an update. We provide more than just jeos in this book.
```

Comment:
The rest is in the "if" above.  And then we must add a statement about proxy--just do it ;)